lib/cell-manager.coffee

```diff
@@ -0,0 +1,51 @@
+_ = require 'lodash'
+
+module.exports = CellManager =
+    removeAllBreakPoints: ->
```

Comment:
`hydrogen:run-breakpoint-cell` sounds a bit odd to me.

An alternative to `cell` would be `code-cell` (used by spyder) or `section`/`code-section` (I think matlab uses this).